Backend/app/provider-platform/dynamic-offer-driver-app/Main/src/SharedLogic/MessageBuilder.hs

```diff
@@ -339,3 +341,19 @@ buildOperatorDeepLinkAuthMessage merchantOperatingCityId req = do
         staticMsg
           & T.replace (templateText "operatorName") req.operatorName
   pure (senderHeader, dynamicMsg)
+
+data BuildFleetLinkSuccessMessageReq = BuildFleetLinkSuccessMessageReq
+  { operatorName :: Text
+  }
+
+buildFleetLinkSuccessMessage ::
+  (EsqDBFlow m r, CacheFlow m r) =>
+  Id DMOC.MerchantOperatingCity ->
+  BuildFleetLinkSuccessMessageReq ->
+  m (Maybe Text, Text)
+buildFleetLinkSuccessMessage merchantOpCityId req = do
```

Comment:
Could you add sms template in migrations for FLEET_LINK_SUCCESS_MESSAGE, something like here

https://github.com/nammayatri/nammayatri/blob/main/Backend/dev/migrations/dynamic-offer-driver-app/0443-add-download-app-key-in-message.sql#L2